### ISSUE TITLE
File "./queue.py", line 175, in get queue.Empty 

### ISSUE THREAD

**Issue by stolinski** [SublimeLinter/SublimeLinter3#433]
On the latest versions of everything. 

```
SublimeLinter: error in SublimeLinter daemon: 
SublimeLinter: -------------------- 
SublimeLinter: Traceback (most recent call last):
  File "/Users/scotttolinski/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/queue.py", line 57, in loop
    item = self.q.get(block=True, timeout=self.MIN_DELAY)
  File "./queue.py", line 175, in get
queue.Empty

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/scotttolinski/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/queue.py", line 65, in loop
    self.lint(view_id, timestamp)
  File "/Users/scotttolinski/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/queue.py", line 111, in lint
    self.callback(view_id, timestamp)
  File "/Users/scotttolinski/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/sublimelinter.py", line 119, in lint
    Linter.lint_view(view, filename, code, hit_time, callback)
  File "/Users/scotttolinski/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/linter.py", line 957, in lint_view
    callback(cls.get_view(vid), linters, hit_time)
  File "/Users/scotttolinski/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/sublimelinter.py", line 167, in highlight
    highlights.draw(other_view)
  File "/Users/scotttolinski/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/highlight.py", line 98, in draw
    all.draw(view)
  File "/Users/scotttolinski/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/highlight.py", line 390, in draw
    self.set_mark_style()
  File "/Users/scotttolinski/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/highlight.py", line 377, in set_mark_style
    self.mark_flags = MARK_STYLES[self.mark_style]
KeyError: ''
```

**Comment by groteworld** [comment]
In your sublimelinter settings file what is your `mark_style` value?

**Comment by stolinski** [comment]
Just the default.

`"mark_style": "outline",`

**Comment by stolinski** [comment]
Found an empty 
`"mark_style": "",`
In the user settings. FYI I didn't modify user or default settings if this is something that's created at install.

**Comment by groteworld** [comment]
Yeah, thats what I was expecting, but I'm not sure why/if that was created
that way. It should match default on first install.

  

Glad this was solved. :beers:

> On Mar 15 2016, at 6:12 pm, Scott Tolinski &lt;notifications@github.com&gt;
wrote:  

>

> Closed [#433](https://github.com/SublimeLinter/SublimeLinter3/issues/433).

>

> —  
You are receiving this because you commented.  
Reply to this email directly or [view it on GitHub](https://github.com/Sublime
Linter/SublimeLinter3/issues/433#event-591028077)![](https://github.com/notifi
cations/beacon/ABWx8m-iJwiJ1tt_-Wuu7Z4xF7YwCpa7ks5pty68gaJpZM4HxdVn.gif)

**Comment by ahennie** [comment]
I had the same issue, and i didn't set it myself. guess it's a bug?
